// [[Reason: TODO File is not auto-fixed. Symbol in computed property not imported.]] ////

//// [tests/cases/compiler/declarationEmitReadonlyComputedProperty.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -7,9 +7,11 @@
 }
 export declare function createInstance(): Interface;
 //# sourceMappingURL=bug.d.ts.map
 //// [index.d.ts]
-export declare const spread: {};
+export declare const spread: {
+    [SYMBOL]: string;
+};
 //# sourceMappingURL=index.d.ts.map
 /// [Errors] ////
 
 index.ts(4,5): error TS1170: A computed property name in a type literal must refer to an expression whose type is a literal type or a 'unique symbol' type.
